src/App.vue

```diff
@@ -41,7 +41,9 @@
   export default class Layer extends Vue {
     protected breakpoints: Dictionary<number> = {
       mobile: 743,
-      desktop: Number.POSITIVE_INFINITY
+      tablet: 1439,
```

Comment:
Related tags in the tablet view.

![image](https://user-images.githubusercontent.com/75546736/219645454-d175a07a-2ae6-49cd-8dd6-c9b56777f78e.png)

Reply:
Things like this happen in multiple sites, like the selected filters.

Reply:
that is ok, as it is the mobile layout